bucket/smartsynchronize.json

```diff
@@ -27,16 +27,16 @@
     },
     "persist": ".settings",
     "checkver": {
-        "url": "https://www.syntevo.com/smartsynchronize/changelog.txt",
-        "regex": "([\\d.]+)"
+        "url": "https://www.syntevo.com/smartsynchronize/download/",
+        "regex": "smartsynchronize-portable-([\\d.]+).7z"
```

Comment:
```suggestion
        "regex": "smartsynchronize-portable-([\\d.]+)\\.7z"
```

Reply:
In this situation, `.` is enough, there should be one character `.`, so regex `.` will match it. Of couse, `\.` could be used, too.

Reply:
There is no need to expect anything when you know there will always be dot literally.